src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -222,14 +224,20 @@ export const NavBar: React.FC<NavBarPropsType> = ({
                   onClick={navigateToHomepage}
                 />
               )}
-              <div
-                className={`nav-party-logo ${
-                  isEventScheduleVisible && "clicked"
-                }`}
-                onClick={toggleEventSchedule}
-              >
-                {navbarTitle} <span className="schedule-text">Schedule</span>
-              </div>
+
+              {shouldShowSchedule ? (
+                <div
+                  className={`nav-party-logo ${
```

Comment:
You can use `classNames` to align this with our latest styles.

Reply:
I was opting for minimal touch, if you enable the ‘hide white space changes’ mode of the PR diff review you’ll see that that’s existing code, indenting just changed because of the conditional rendering I added.

Guven urgency and workload of GitHub features needed, I don’t have bandwidth to spend time polishing existing tech debt (unfortunately)

Reply:
[OOS] Unrelated tech debt, future cleanup.